cornflow_client/tests/const.py

```diff
@@ -79,3 +86,20 @@
         ]
     }
 )
+
+PULP_EXAMPLE = _get_file("./data/pulp_example_data.json")
+
+PUBLIC_DAGS = [
+    "solve_model_dag",
+    "graph_coloring",
+    "timer",
+    "bar_cutting",
+    "facility_location",
+    "graph_coloring",
+    "hk_2020_dag",
+    "knapsack",
+    "roadef",
+    "rostering",
+    "tsp",
+    "vrp",
+]
```

Comment:
are you going to update this list as the public repo is updated? Sounds ambitious.

Reply:
This list is the list of the current dags on the public repo. The test checks that at least these dags are deployed, so if add dags to the public repo the test here would still pass.

The only problem would be if we change the name of a dag in the public repo or if we delete one of them, cases that I think are going to be rare, but anyway, I will try to think of any other solution.